src/index.html

```diff
@@ -11,6 +11,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="main">
+      <form class="search__bar content__form" method="get"
+      data-qa="big">
+        <label class="search__label" for="search__bar">
+        </label>
+        <input class="search__input" type="text"
+      placeholder="Try &quot;Los Angeles&quot;" data-qa="keypress"
+      id="search__bar">
+      </form>
+      <form class="search__bar search__bar-small content__form" method="get"
```

Comment:
please refactor both form
```suggestion
      <form 
        class="search__bar content__form" 
        method="get"
        data-qa="big"
      >
        <label class="search__label" for="search__bar"></label>
        <input 
          class="search__input" type="text"
          placeholder="Try &quot;Los Angeles&quot;" 
          data-qa="keypress"
          id="search__bar"
        >
      </form>
      
      <form 
        class="search__bar search__bar-small content__form" 
        method="get"
```

Reply:
Refactored